templates/handlers.js

```diff
@@ -1,9 +1,18 @@
 const { Context } = require('./boltzmann.js') // optionally pull in typescript definition
 
 greeting.route = 'GET /hello/:name'
+// {% if templates %}
+async function greeting(/** @type {Context} */ context) {
+  return {
+    [Symbol.for('template')]: 'index.html',
```

Comment:
Should we mount `greeting` at `GET /*`, so users opening `localhost:5000` for the first time see it (vs. a 404)?